### PR TITLE
Monero for Merchants Turkish translation

### PR DESCRIPTION
**Pull request by furkankir**
Turkish translation has been made along with a README in the "tr" folder.

These changes has been proofread by me who is a native turkish speaker

### CODE REVIEW

**Comment by ASchmidt1024** [comment]
Great. I'll get to work on that shortly.

**Review by ASchmidt1024 — APPROVED** [review]
🚀 Yeah, yeah, yeah!

File: translations/tr/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Tüccarlar için Monero
+
+"Monero kabulünü artırmak için tasarlanmış bir broşürdür."
+
+[&#10515;PDF İndirin](./Monero%20for%20Merchants%20Booklet-turkish.pdf)
```
Comment (HardenedSteel) [inline translations/tr/README.md:5]: PDF'i indirin

File: translations/tr/translation-turkish.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+---
+
+Avantajlar
+
+İşiniz için
+
+**Hızlı işlemler**: Ödemeleri neredeyse gerçek zamanlı olarak alın.
+
+**Düşük kesintiler**: Banka veyahut kredi kartı kesintileri yok.
```
Comment (HardenedSteel) [inline translations/tr/translation-turkish.md:65]: Veya kullanılması daha uygun olur diye düşünüyorum

**Review by HardenedSteel — COMMENTED** [review]
Bazı eksikliklerin giderilmesi gerekiyor.

**Pull request merged**